pkg/apis/pipeline/v1beta1/taskrun_types.go

```diff
@@ -255,7 +255,7 @@ type TaskRunStatusFields struct {
 	// Deprecated: this field is not populated and is preserved only for backwards compatibility
 	// +optional
 	// +listType=atomic
-	ResourcesResult []RunResult `json:"resourcesResult,omitempty"`
+	ResourcesResult []PipelineResourceResult `json:"resourcesResult,omitempty"`
```

Comment:
why did you change back to `PipelineResourceResult`?

Reply:
I thought that using `PipelineResourceResult` could keep the previous reference for `ResourcesResult` which uses `PipelineResourcesResult` and we wouldn't need to import the newly created result package here. But if it made it more confusing I would be happy to change it back.